haystack/utils/import_utils.py

```diff
@@ -5,12 +5,54 @@
 import zipfile
 import requests
 import logging
+import importlib
 from pathlib import Path
 
 
 logger = logging.getLogger(__name__)
 
 
+def safe_import(import_path, classname, dep_group):
+    """
+    Method that allows the import of "non-existing" document stores.
+    Doc stores can be installed one by one with extras_require (see setup.cfg)
+    but they need to be all imported in haystack.document_stores.__init__()
+    
+    Therefore, in case of an ImportError, the class to import is replaced by
+    a hollow MissingDependency function, which will throw an error when
+    inizialized.
+    """
+    try:
+        module = importlib.import_module(f"{import_path}")
```

Comment:
What's the string interpolation doing here? Do we expect other types than `str` for `import_path`?

Reply:
Good question indeed :sweat_smile: A leftover for sure